apps/io-wallet-user-func/src/infra/azure/queue/wallet-instance-creation.ts

```diff
@@ -0,0 +1,41 @@
+import { QueueClient } from "@azure/storage-queue";
+import * as E from "fp-ts/lib/Either";
+import * as J from "fp-ts/lib/Json";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import { WalletInstance } from "io-wallet-common/wallet-instance";
+
+export interface IWalletInstanceCreationQueue {
+  insert: (walletInstance: WalletInstance) => TE.TaskEither<Error, void>;
+}
+
+export class WalletInstanceCreationStorageQueue
```

Comment:
since `wallet-instance-revocation.ts` has the same code, how about we standardize them and use a single class?

Reply:
We could:
- delete one between the two class/interfaces (it's indifferent, the code is the same anyway!)
- rename the remaining class/interface to `WalletInstanceStorageQueue`
- use the renamed class/interface in both the revocation and creation process, obviously passing the correct data (storage queue name) during instantiation

Reply:
yes you could rename it to WalletInstanceQueue and use that